doc/newsboat.asciidoc

```diff
@@ -1306,6 +1306,32 @@ open dialogs by selecting them and pressing "Ctrl-X".
 
 include::chapter-podcasts.asciidoc[]
 
+=== Media Content
```

Comment:
After looking at the table of contents, I also feel that this new chapter could be named better. For example, "Podcast Support" is well-named because a person who never used Newsboat can immediately tell what it's about, and decide if it's relevant to them. "Macro Support" isn't as well-named, because "macro" is a Newsboat-specific feature; but fixing that is outside of scope for this PR. "Media Content" isn't very well-named either because it's really hard to tell what this chapter will describe; one has to skim it to figure that out.

Does that make sense to you? I'm thinking of something along the lines of "Handling media content with macros", but I'm open to suggestions.

Reply:
I believe the new arrangement and naming will make it easy to find relevant chapters.